server/php-worker.cpp

```diff
@@ -127,7 +139,7 @@ void PhpWorker::state_try_start() noexcept {
 }
 
 void PhpWorker::state_init_script() noexcept {
-  double timeout = finish_time - precise_now - 0.01;
```

Comment:
Here is potentially dangerous place for job workers. Before that pr every timeout was less on 10ms